rm/src/lib.rs

```diff
@@ -122,16 +124,38 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
                 .join("namespaces")
                 .join(&urn.encode_id());
 
-            if !options.confirm
-                || term::confirm(format!(
+            if options.confirm
+                && !term::confirm(format!(
                     "Are you sure you would like to delete {}?",
                     term::format::dim(namespace.display())
                 ))
             {
-                rad_untrack::execute(urn, None, rad_untrack::Options { peer: None }, &profile)?;
-                fs::remove_dir_all(namespace)?;
-                term::success!("Successfully removed project {}", &urn);
+                return Ok(());
             }
+
+            let all_untracked = tracking::untrack_all(
+                &storage,
+                urn,
+                tracking::UntrackAllArgs {
+                    policy: tracking::policy::UntrackAll::Any,
+                    prune: true,
+                },
+            )?;
+            fs::remove_dir_all(namespace)?;
+
+            if let Ok((cwd_urn, cwd_repo)) = project::cwd() {
+                if cwd_urn == *urn {
+                    for p in all_untracked.untracked.flatten() {
+                        let err =
+                            term::remote::remove(&p.remote.to_string(), &storage, &cwd_repo, urn);
+                        if let Err(err) = err {
+                            eprintln!("unable to remove git remote '{}': {}", p.remote, err);
```

Comment:
Is this expected to fail often? And is the whole removal operation considered errornous if only one of the remotes can't be removed? If so, we could return the error here already such that it bubbles up correctly. Otherwise I'd suggest to collect the remotes that could not be removed and show an error / warning listing these.